tools/test_value_scraper_vmc.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python
+"""
+This is a helper script to build the test values that we need.
+Currently, we lack the ability to actually spin up a VMC
+of our own design, specifying all the values that we desire.
+
+This script will update tests/integration/vmc_config containing
+different sections
+for vmc nsx section is vmc_nsx_connect,
+
+Following field are required for this
+
+authorization_host it cloud console
+authorization_host = console-stg.cloud.vmware.com
+developer API token
+refresh_key = CHANGE ME
+Organization_id
+org_id = 10e1092f-51d0-473a-80f8-137652fd0c39
+SDDC Id
+sddc_id = ef562ce8-819e-47dc-9353-2ef0366bc9d1
+
+other variable in these section will be populated by this script
+and will update that config with appropriate values for use in the
+integration suite.
+
+It's not an ideal way to test, but it does at least provide some automation
+to the process.
+ """
+import argparse
+import json
+import sys
+import urllib.parse
+from pathlib import Path
+
+import saltext.vmware.modules.vmc_sddc as vmc_sddc
+
+vmc_config_dict = {
```

Comment:
**suggestion** this dictionary should not be here. Moving the values from the `.ini` file to code does not solve the problem of having this information in the repo.

Information should come from two places:
1) provided by the user
2) pulled from vSphere

And *neither* of those pieces should *ever* be tracked in the repository. That's just asking someone to upload their information to the Internet, leading to their infrastructure turning into bitcoin miners or worse.

The only information that should be in the repository is example information that is **necessary** to help the user see the shape of the information they need to provide. 

For instance, if you're asking for a "host", you could say something like:

> host: example.com or 203.0.113.3

And then the user will understand a hostname or IP address is sufficient. But you should never put your *actual* IP address in there. Same thing for email addresses, etc. etc.

The only exception to this rule is if there exist default values that are well known, e.g. a raspberry pi's default password is `pi/raspberry`.

Reply:
Updated script as per suggestion.